### PR TITLE
Added env variable and path for latest tool versions

### PR DESCRIPTION
**Pull request by yenicapotediaz**
this covers https://github.com/samsung-cnct/k2-tools/issues/49

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,6 +20,11 @@ ENV     K8S_HELM_VERSION_1_4=v2.1.3
 ENV     K8S_HELM_VERSION_1_5=v2.3.1
 ENV     K8S_HELM_VERSION_1_6=v2.4.2
 
+#Latest version of tools
+ENV     LATEST=v1.6
+ENV     K8S_VERSION_LATEST=K8S_VERSION_1_6
```
Comment (joejulian) [inline Dockerfile]: Did you mean ...`=$K8S_`...

Comment (yenicapotediaz) [inline Dockerfile]: yes, yes I do...

File: Dockerfile [hunk 2]
```diff
@@ -12,9 +12,6 @@ ENV     CLOUDSDK_PYTHON_SITEPACKAGES 1
         # google cloud kubectl is superceeded by downloaded kubectl
 ENV     PATH $PATH:/google-cloud-sdk/bin
 
-ENV     K8S_VERSION=v1.4.12
-ENV     K8S_HELM_VERSION=v2.3.1
```
Comment (coffeepac) [inline Dockerfile]: need to keep these until both helm and kubectl aren't using a binary that is in $PATH

Comment (yenicapotediaz) [inline Dockerfile]: Got it

File: Dockerfile [hunk 3]
```diff
@@ -83,19 +88,13 @@ RUN     sed -i -- 's/\"disable_updater\": false/\"disable_updater\": true/g' /go
 
 # Kubernetes
 
-        # Adding Kubernetes
-RUN     wget https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod a+x kubectl && mv kubectl /usr/bin
```
Comment (coffeepac) [inline Dockerfile]: also need to install them in the image.

Comment (yenicapotediaz) [inline Dockerfile]: Yep, I'll add that back.

Comment (joejulian) [inline Dockerfile]: Shouldn't this just be symlinks rather than a complete other copy of these big files?

File: Dockerfile [hunk 4]
```diff
@@ -23,6 +23,11 @@ ENV     K8S_HELM_VERSION_1_4=v2.1.3
 ENV     K8S_HELM_VERSION_1_5=v2.3.1
 ENV     K8S_HELM_VERSION_1_6=v2.4.2
 
+#Latest version of tools
+ENV     LATEST=v1.6
+ENV     K8S_VERSION_LATEST=$K8S_VERSION_1_6
+ENV     K8S_HELM_VERSION_LATEST=$K8S_HELM_VERSION_1_6
+
```
Comment (guineveresaenger) [inline Dockerfile]: why is there both LATEST and K8S_VERSION_LATEST?

Comment (yenicapotediaz) [inline Dockerfile]: LATEST is used in the path on line 101

**Comment by guineveresaenger** [comment]
General question: this means we can just pull `/opt/cnct/kubernetes/latest` in k2, rather than running a latest_version filter? if so, I should make a ticket for that

File: Dockerfile (hunk 4 above)
Comment (guineveresaenger) [inline Dockerfile]: sure but why not use K8S_VERSION_LATEST ?

**Comment by yenicapotediaz** [comment]
@guineveresaenger Yes

File: Dockerfile (hunk 4 above)
Comment (coffeepac) [inline Dockerfile]: K8S_VERSION_LATEST is deprecated and should be deleted when a couple of other k2cli issues are resolved.

Comment (guineveresaenger) [inline Dockerfile]: ah! thank you for explaining.

**Pull request merged**